airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -611,6 +613,14 @@ private void readSsl(final JsonNode sslMethod, final List<String> additionalPara
     }
   }
 
+  @Override
+  public Collection<AutoCloseableIterator<AirbyteMessage>> readStreams(JsonNode config, ConfiguredAirbyteCatalog catalog, JsonNode state)
```

Comment:
why do we need to override both readStreams and logPreSyncDebugData? From what I can tell, super.readStreams calls this.read(), which calls logPreSyncDebugData. Am I missing something here?

Reply:
That's right. Thanks!